api/index.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
 require __DIR__.'/src/App/App.php';
-
+error_reporting(E_ALL);
+set_error_handler(function ($severity, $message, $file, $line) {
+    if (error_reporting() & $severity) {
+        throw new \ErrorException($message, 0, $severity, $file, $line);
+    }
+});
```

Comment:
Ok I am going to be picky...

I would love all changed in a given commit to be related.  This seems valid and all but is not directly related to the store code later. 

So my gut says a different commit maybe different pull request.